config/examples/Tevo/Tarantula Pro/Configuration.h

```diff
@@ -23,6 +23,8 @@
 #error "Don't build with import-2.1.x configurations!"
 #error "Use the 'bugfix...' or 'release...' configurations matching your Marlin version."
 
+//#define MOTHERBOARD BOARD_MKS_SGEN_L  // Uncomment for the MKS SGEN-L board (32-bit), otherwise MKS GEN-L board (8-bit) is assumed
```

Comment:
I wish we had a way to do this that was friendlier for the Custom Configuration Editor in Auto Build Marlin. We'll have to hire a board of consultants and a staff of Ph.D. programmers to sort this one out.

Reply:
Indeed. True config parsing would be slick, especially for forks I maintain that support multiple boards.